### ISSUE TITLE
Flickering in Mac OS Safari v 16.2

### ISSUE THREAD

**Issue by tnog** [sneas/img-comparison-slider#110]
Testing on your [examples page](https://img-comparison-slider.sneas.io/examples.html), I've been able to reproduce the issue. 

Encountered the issue on both my local machine: Monterey: 12.6.2 / Safari 16.2

Also tested on Browserstack Ventura / Safari 16.0 (see attached screen recording): 

https://user-images.githubusercontent.com/2049299/218229465-3b99fafe-2d73-42c5-909e-d59fbdb312a2.mp4

The issue seems only to impact Safari, but I've been also able to reproduce it on older versions of Mac OS like Big Sur/ Safari 14.1

**Comment by sneas** [comment]
Hi @tnog, thank you for the issue and video!

The problem with text selection while sliding in Safari has been fixed as v7.8.2 of the component.

While trying to resolve the performance issue in Safari, I noticed a peculiar behaviour: the component works way faster (not ideal, though) when the default on-focus outline is disabled or reset with a custom value.


https://user-images.githubusercontent.com/144651/218315290-e82d62c1-036b-43e0-afc8-ff41f725afc8.mov
